Sources/XcodeGraph/Models/Project.swift

```diff
@@ -202,6 +291,51 @@ public struct Project: Hashable, Equatable, CustomStringConvertible, CustomDebug
             )
         }
 
+        public static func test(
+            path: AbsolutePath = try! AbsolutePath(validating: "/Project"), // swiftlint:disable:this force_try
+            sourceRootPath: AbsolutePath = try! AbsolutePath(validating: "/Project"), // swiftlint:disable:this force_try
+            // swiftlint:disable:next force_try
+            xcodeProjPath: AbsolutePath = try! AbsolutePath(validating: "/Project/Project.xcodeproj"),
+            name: String = "Project",
+            organizationName: String? = nil,
+            classPrefix: String? = nil,
+            defaultKnownRegions: [String]? = nil,
+            developmentRegion: String? = nil,
+            options: Options = .test(automaticSchemesOptions: .disabled),
+            settings: Settings = Settings.test(),
+            filesGroup: ProjectGroup = .group(name: "Project"),
+            targets: [Target] = [Target.test()],
+            packages: [Package] = [],
+            schemes: [Scheme] = [],
+            ideTemplateMacros: IDETemplateMacros? = nil,
+            additionalFiles: [FileElement] = [],
+            resourceSynthesizers: [ResourceSynthesizer] = [],
+            lastUpgradeCheck: Version? = nil,
+            type: ProjectType
+        ) -> Project {
+            Project(
+                path: path,
+                sourceRootPath: sourceRootPath,
+                xcodeProjPath: xcodeProjPath,
+                name: name,
+                organizationName: organizationName,
+                classPrefix: classPrefix,
+                defaultKnownRegions: defaultKnownRegions,
+                developmentRegion: developmentRegion,
+                options: options,
+                settings: settings,
+                filesGroup: filesGroup,
+                targets: targets,
+                packages: packages,
+                schemes: schemes,
+                ideTemplateMacros: ideTemplateMacros,
+                additionalFiles: additionalFiles,
+                resourceSynthesizers: resourceSynthesizers,
+                lastUpgradeCheck: lastUpgradeCheck,
+                type: type
+            )
+        }
+
         public static func empty(
```

Comment:
Should we a) deprecate it as well b) don't consider test helpers as part of the public interface?

I'd probably deprecate, so we don't cause a breaking change for `tuist/tuist`